db/nft.go

```diff
@@ -373,24 +385,36 @@ func GetCollector(conn *pgxpool.Conn, q QueryCollectorRequest, p PageRequest) (r
 			'count', count)),
 		COUNT(*) OVER() AS row_count
 	FROM (
-		SELECT n.owner, i.iscn_id_prefix, c.class_id, SUM(%s.latest_price) AS value, COUNT(DISTINCT n.id) as count
+		SELECT n.owner, i.iscn_id_prefix, c.class_id, SUM(%[1]s) AS value, COUNT(DISTINCT n.id) as count
 		FROM iscn AS i
 		JOIN iscn_latest_version
 		ON i.iscn_id_prefix = iscn_latest_version.iscn_id_prefix
 			AND ($5 = true OR i.version = iscn_latest_version.latest_version)
 		JOIN nft_class AS c ON i.iscn_id_prefix = c.parent_iscn_id_prefix
 		JOIN nft AS n ON c.class_id = n.class_id
 			AND ($4::text[] IS NULL OR cardinality($4::text[]) = 0 OR n.owner != ALL($4))
+		JOIN (
+			SELECT nft_id, receiver, MAX(id) AS max_id
+			FROM nft_event
+			WHERE price IS NOT NULL
+			GROUP BY nft_id, receiver
+		) AS latest_e 
```

Comment:
```suggestion
  JOIN LATERAL (
    SELECT nft_id, receiver, MAX(id) AS max_id
    FROM nft_event
    WHERE price IS NOT NULL
      AND nft_id = n.nft_id
      AND receiver = n.owner
    GROUP BY nft_id, receiver
  ) AS latest_e 
```
Try this, amazingly fast with index (while horribly slow without index)

Reply:
Yes, deployed to Rinkeby and really amazingly fast, much thanks!!